src/lib.rs

```diff
@@ -230,3 +261,59 @@ pub fn locate_dsym<T>(path: T, uuid: Uuid) -> Result<PathBuf, Error>
 {
     dsym::locate(path.as_ref(), uuid)
 }
+
+/// Attempt to locate the separate debug symbol file for the object file at `path` with
+/// build ID `id`.
+pub fn locate_debug_build_id(id: &[u8]) -> Result<PathBuf, Error> {
+    if id.len() < 2 {
+        return Err(failure::err_msg("Build ID is too short"));
+    }
+
+    // Try "/usr/lib/debug/.build-id/12/345678etc.debug"
+    let mut f = format!("/usr/lib/debug/.build-id/{:02x}/", id[0]);
```

Comment:
It might be nice to allow callers to pass a path or set of paths to use instead of hardcoding /usr/lib/debug, but this is likely to be what most people want anyway.